proto/provenance/expiration/v1/expiration.proto

```diff
@@ -0,0 +1,34 @@
+// Since: provenance 1.12
+syntax = "proto3";
+package provenance.expiration.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package          = "github.com/provenance-io/provenance/x/expiration/types";
+option java_package        = "io.provenance.expiration.v1";
+option java_multiple_files = true;
+
+// Params defines the set of params for the expiration module.
+message Params {
+  option (gogoproto.equal)            = false;
+  option (gogoproto.goproto_stringer) = false;
+
+  // the deposit amount for holding module assets on chain
+  cosmos.base.v1beta1.Coin deposit = 2 [(gogoproto.nullable) = false];
+}
+
+// Expiration holds a typed key/value structure for data associated with an expiring module asset
+message Expiration {
+  // the module asset identifier
+  string module_asset_id = 1;
+  // The bech32 address the expiration is bound to
+  string owner = 2;
+  // The block height the module asset expires
+  int64 block_height = 3;
+  // The deposit amount held while module asset is in use
+  cosmos.base.v1beta1.Coin deposit = 4 [(gogoproto.nullable) = false];
+  // Messages relating to the expiring module asset
+  repeated google.protobuf.Any messages = 5;
```

Comment:
Are messages going to be `sdk.Msg` types once parses? Similar to what the `v1/gov` module does now in `v0.46`?

https://github.com/cosmos/cosmos-sdk/blob/a1143138716b64bc4fa0aa53c0f0fa59eb675bb7/x/gov/types/v1/msgs.go#L38-L40